start.py

```diff
@@ -1,40 +1,100 @@
-from board import Board, EndStates, SpotStates, BoardState, WinningCombos 
+from board import Board, EndStates, SpotStates, BoardState, WinningCombos
```

Comment:
I think it might be a good idea to revisit the classes you have here knowing what you now know about classes and seeing if the stuff in there makes sense.

Reply:
+1